index.js

```diff
@@ -3,5 +3,5 @@
 var navigator = require('global/window').navigator
 
 module.exports = typeof navigator !== 'undefined' &&
```

Comment:
The regular expression is flagged as case insensitive (`/i`). Fine with lowercasing the input to avoid the need for a flag but it should be one or the other.

Reply:
Note the use of `global/window` to access navigator, which is then used in tests. Replacing this with `window` is not correct.